bigtable/client/table_sample_row_keys_test.cc

```diff
@@ -51,6 +79,8 @@ TEST_F(TableSampleRowKeysTest, SimpleVectorTest) {
   EXPECT_CALL(*reader, Finish()).WillOnce(Return(grpc::Status::OK));
   auto result = table_.SampleRows<std::vector>();
   auto it = result.begin();
+  EXPECT_EQ(typeid(std::vector<bigtable::v0::Table::RowKeySample>),
```

Comment:
Or you could say 

```C++
std::vector<bigtable::Table::RowKeySample> result = ...
```

which would detect the problem at compile time.

Reply:
Done.